README.md

```diff
@@ -100,7 +103,39 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+class Animal
+
+  attr_accessor :name
+
+  def initialize(name)
+    @name = name
+  end
+
+  def greet
+    puts "Hi"
+  end
+
+end
+
+pumba = Animal.new("Pumba")
+
+class Lion < Animal
+  @@pack = 0
+  def initialize(name)
+    @name = name
+    @@pack += 1
+  end
+
+  def king
```

Comment:
This works but it'd be better if you could take care of this in `initialize` when the instance is being generated.